### DISCUSSION TITLE
Expo icon not updating in Play Store.

### DISCUSSION THREAD

**Discussion by anujdivesh** [expo/expo#16120] [Archived: Questions]
I have recently updated my app icon. It changes in Apple store but not in google store.

**Comment by jurica-penjgusic** [comment]
Maybe changing icon directly in Google store in app information section helps.

**Reply by anujdivesh** [reply]
@jurica-penjgusic  So the app icon is updated in google play console also in the bundle build, but when i download the app from the store, it does not have the updated icon. Works fine for Apple store,
